lib/pr_checker.js

```diff
@@ -230,6 +238,25 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commits, reviews, logger) {
```

Comment:
Hmm wait actually, we don't need to pass them here, those are supposed to get assigned in the constructor